projects/plugins/social/src/js/editor.js

```diff
@@ -67,6 +68,7 @@ const JetpackSocialSidebar = () => {
 
 	return (
 		<PostTypeSupportCheck supportKeys="publicize">
+			<SyncPostDataToStore />
```

Comment:
We could use the hook directly here, as there are no conditionals. perhaps if we could reorganise the code in the Jetpack version so that the Publicize sidebar panels form their own component that is not rendered if the module is loading/disabled, then we could avoid the conditional hook problem.

Reply:
That will need some refactor which we can do in a separate PR. Then we can get rid of that component and use the hook directly.